index.html

```diff
@@ -5495,9 +5496,9 @@ <h3>Base Context</h3>
 credential, such as resources that are linked to via URLs, are not
 cryptographically protected by default. It is considered a best practice to
 ensure that the same sorts of protections are provided for any URL that is
-critical to the security of the credential through the use of permanently cached
-files and/or cryptographic hashes. See the <a
-data-cite="vc-imp-guide/#content-integrity">Content Integrity</a>
+critical to the security of the <a>verifiable credential</a> through the use of
+permanently cached files and/or cryptographic hashes. See the
```

Comment:
are these files intended to be immutable?
if not, is there a TTL that informs caching behavior?

Reply:
Once the VCWG takes the spec to REC, the files are intended to be immutable (modulo the errata statement @OR13 mentions below https://github.com/w3c/vc-data-model/pull/1249#pullrequestreview-1587450604).